build.install4j

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<install4j version="6.1.2" transformSequenceNumber="5">
+<install4j version="6.1.5" transformSequenceNumber="5">
```

Comment:
That change only updates the install4j Gradle plugin--it does not update install4j itself.

The Gradle build uses whatever install4j installation your `install4jHomeDir` project variable is set to.  In the Travis build, it points to install4j 6.1.2 downloaded and installed from the `triplea-game/assets` repo.  See [`.travis/setup`](https://github.com/triplea-game/triplea/blob/master/.travis/setup) for the details.

I'm going to submit the PR to put install4j 6.1.5 into the `assets` repo now.  We may just be able to use this PR to upgrade the entire build to 6.1.5.  That way we don't have to do 6.1.5 testing twice. :smiley: